ddtrace/internal/schema/__init__.py

```diff
@@ -18,10 +20,13 @@ def _validate_schema(version):
 
 
 __schema_version = os.getenv("DD_TRACE_SPAN_ATTRIBUTE_SCHEMA", default="v0")
+__remove_client_service_names = asbool(os.getenv("DD_TRACE_REMOVE_CLIENT_SERVICE_NAMES_ENABLED", default=False))
 _validate_schema(__schema_version)
 
-DEFAULT_SPAN_SERVICE_NAME = _DEFAULT_SPAN_SERVICE_NAMES[__schema_version]
-schematize_service_name = _SPAN_ATTRIBUTE_TO_FUNCTION[__schema_version]["service_name"]
+__service_name_schema_version = "v0" if __schema_version == "v0" and not __remove_client_service_names else "v1"
```

Comment:
Nit: Is v0 referenced in a few places? Should it have a constant?

Reply:
probably, I can do a refactor in a follow-up PR, I think it'd muddy this one too much